collector/odigosotelcol/components.go

```diff
@@ -140,6 +142,8 @@ func components() (otelcol.Factories, error) {
 		datasetexporter.NewFactory(),
 		dynatraceexporter.NewFactory(),
 		elasticsearchexporter.NewFactory(),
+		// TODO we need to create this exporter
+		// quickwitexporter.NewFactory(),
```

Comment:
@fmassot Here, we need to make this exporter available. What do you think about-it ? Thanks

Reply:
Apparently, just need to use `otlpexporter` (cc @fmassot)